dropwizard-metrics/src/test/scala/org/http4s/metrics/dropwizard/DropwizardServerMetricsSpec.scala

```diff
@@ -169,7 +169,8 @@ class DropwizardServerMetricsSpec extends Http4sSpec {
       count(registry, Counter("server.default.active-requests")) must beEqualTo(0)
       count(registry, Timer("server.default.requests.total")) must beEqualTo(1)
       valuesOf(registry, Timer("server.default.requests.headers")) must beSome(Array(50000000L))
-      valuesOf(registry, Timer("server.default.get-requests")) must beSome(Array(100000000L))
+      // Not sure why this changed
```

Comment:
Looking at the implementation, I'm surprised that it wasn't always so. But the second and third calls are close enough together to be immaterial in the real world. I don't think we should hold the release hostage figuring this out.